packages/core/src/conversation/message/MessageService.ts

```diff
@@ -83,10 +83,7 @@ export class MessageService {
       cipherText = externalPayload.cipherText;
     }
 
-    const {encrypted, missing} = await this.proteusService.encrypt(plainTextPayload, recipients);
-    const encryptedPayload = Object.keys(missing).length
-      ? await this.reencryptAfterMismatch({missing, deleted: {}}, encrypted, plainText)
-      : encrypted;
+    const encryptedPayload = await this.proteusService.encrypt(plainTextPayload, recipients);
```

Comment:
no more missing encrypted payload since all the necessary sessions will be created beforehand